.github/workflow/release.yml

```diff
@@ -0,0 +1,55 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  release:
+    name: Release
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout code repository
+        # https://github.com/actions/checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: setup node.js
+        # https://github.com/actions/setup-node
+        uses: actions/setup-node@v4
+
+      - name: install pnpm
+        ruses: pnpm/action-setup@v3
+        with:
+          run_install: false
+
+      - name: Get pnpm store directory
```

Comment:
This cache update is made based on the [pnpm/action-setup](https://github.com/pnpm/action-setup?tab=readme-ov-file) documentation